src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AutoDisenchanter.java

```diff
@@ -141,8 +141,7 @@ private void transferEnchantments(ItemStack item, ItemStack book, Map<Enchantmen
             // Prevent future enchantment duplication
             if (wasEnchantmentRemoved && !stillHasEnchantment) {
                 meta.addStoredEnchant(entry.getKey(), entry.getValue(), true);
-            }
-            else {
+            } else {
                 //                                                                                                      Get Enchantment Name
```

Comment:
```suggestion
                // Get Enchantment Name
```
I don't think these spaces were intended...

Reply:
Yep, my bad.